### PR TITLE
SOV-1906: Constants refactoring

### PR DESCRIPTION
**Pull request by tiltom**
Resolves https://sovryn.atlassian.net/browse/SOV-1906

### CODE REVIEW

File: apps/frontend/src/constants/gasLimits.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  ADJUST_TROVE: 4000000,
+  CLOSE_TROVE: 350000,
+  CLOSE_DLLR_TROVE: 500000,
+  CONVERT: 200000,
+  STABILITY_POOL: 250000,
+  STABILITY_POOL_INC_WITHDRAW: 490000,
+  STABILITY_POOL_DLLR: 500000,
+  STABILITY_POOL_DLLR_INC_WITHDRAW: 690000,
+  REWARDS: 240000,
+  TRANSFER_LOC: 900000,
```
Comment (soulBit) [inline apps/frontend/src/constants/gasLimits.ts]:
```suggestion
  MAX: 6_800_000,
  OPEN_TROVE: 1_100_000,
  ADJUST_TROVE: 4_000_000,
  CLOSE_TROVE: 350_000,
  CLOSE_DLLR_TROVE: 500_000,
  CONVERT: 200_000,
  STABILITY_POOL: 250_000,
  STABILITY_POOL_INC_WITHDRAW: 490_000,
  STABILITY_POOL_DLLR: 500_000,
  STABILITY_POOL_DLLR_INC_WITHDRAW: 690_000,
  REWARDS: 240_000,
  TRANSFER_LOC: 900_000,
```

File: apps/frontend/src/constants/infrastructure.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    'https://rsk-live.sovryn.app/rpc',
+    'https://public-node.rsk.co',
+  ],
+  [Environments.Testnet]: [
+    'https://testnet.sovryn.app/rpc',
+    'https://public-node.testnet.rsk.co',
+  ],
+};
+
+export const RSK_EXPLORER = {
```
Comment (soulBit) [inline apps/frontend/src/constants/infrastructure.ts]:
```suggestion

//TODO: refactor this into separate dictionary file once we add more chains
export const RSK_EXPLORER = {
```

File: apps/frontend/src/constants/general.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  MYNT: process.env.REACT_APP_GRAPH_MYNT,
+};
+
+export const EXPORT_RECORD_LIMIT = 500;
+export const DEFAULT_HISTORY_FRAME_PAGE_SIZE = 10;
+
+export const DEFAULT_TIMEOUT_SECONDS = 5000;
+export const LIQUIDATION_RESERVE_AMOUNT = 20;
+
+export const BTC_IN_SATOSHIS = 100000000;
```
Comment (soulBit) [inline apps/frontend/src/constants/general.ts]:
```suggestion
export const BTC_IN_SATOSHIS = 100_000_000;
```

File: apps/frontend/src/constants/currencies.ts [hunk 4]
```diff
@@ -0,0 +1,2 @@
+export const BITCOIN = 'BTC';
+export const USD = 'USD';
```
Comment (soulBit) [inline apps/frontend/src/constants/currencies.ts:2]: Extract the `XX_RENDER_PRECISION` constants from `apps\frontend\src\app\3_organisms\ZeroLocForm\constants.ts` here as well, they are used in other places than just LoC form

**Review by soulBit — CHANGES_REQUESTED** [review]

**Review by creed-victor — APPROVED** [review]

**Review by soulBit — APPROVED** [review]

**Pull request merged**